addons/dialogue_nodes/objects/DialogueParser.gd

```diff
@@ -358,6 +358,10 @@ func _parse_variable_names(value: String) -> Array:
 # FIXME : Length calculation is borked when the value has [, ] unrelated to any bbcodes.
 # Updates all the [wait] bbcode tags in the given text to include additional info about the text
 func _update_wait_tags(node: RichTextLabel, value: String) -> String:
+	# Empty string freezes dialogues
+	if value == "":
+		value = " "
```

Comment:
Small nit: the plugin uses single quotes ( ' ) instead of double quotes ( " ) everywhere. So please replace the double quotes here.

Reply:
Fixed!